crates/chain/src/local_chain.rs

```diff
@@ -71,18 +64,21 @@ impl LocalChain {
         }
     }
 
+    /// Get a reference to the inner map of block height to hash.
+    pub fn inner(&self) -> &BTreeMap<u32, BlockHash> {
```

Comment:
just call this `blocks` it's not really a wrapper so `inner` doesn't make much sense.